web/satellite/src/components/project/UsageReport.vue

```diff
@@ -88,21 +88,19 @@ import { toUnixTimestamp } from '@/utils/time';
                 Datepicker,
             },
             beforeMount: function() {
-                const currentDate = new Date();
-                const previousDate = new Date();
-                previousDate.setDate(1);
+				const currentDate = new Date();
 
-                this.$data.dateRange.startDate = previousDate;
+                this.$data.dateRange.startDate = new Date(currentDate.getFullYear(), currentDate.getMonth(), 1);
                 this.$data.dateRange.endDate = currentDate;
             },
             beforeRouteLeave: function(to, from, next) {
-                const currentDate = new Date();
-                const previousDate = new Date();
-                previousDate.setDate(1);
+				const currentDate = new Date();
 
-                this.$data.dateRange.startDate = previousDate;
+                this.$data.dateRange.startDate = new Date(currentDate.getFullYear(), currentDate.getMonth(), 1);
```

Comment:
out of curiosity, how does this work with timezones ?

Reply:
good question. But I guess it's fine, cause these are not invoices it just shows usage relatively to user timezone